src/quo2/components/list_items/account/view.cljs

```diff
@@ -8,6 +8,14 @@
             [reagent.core :as reagent]
             [quo2.components.icon :as icon]))
 
+(defn- network-view
+  [network]
+  [text/text
+   {:size   :paragraph-2
+    :weight :regular
+    :style  {:color (colors/custom-color network)}}
+   (str (subs (name network) 0 3) (when (= network :arbitrum) "1") ":")])
```

Comment:
perhaps this should just take a text-value prop and we do this on the application side? i.e in status-im2 ns

Reply:
@J-Son89 I have updated it to pass the short name of the network